StoreApp.BLL/Services/OrderDetailService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using System.Collections.Generic;
+
+namespace StoreApp.BLL.Services
+{
+    class OrderDetailService : IOrderDetailService
+    {
+        IUnitOfWork Database;
+        public OrderDetailService(IUnitOfWork eof)
```

Comment:
I noticed you changed the name of the parameter. It's a minor point but sometimes it's easy to read code if you use the same approaches.

Reply:
fixed